check-ecs-exec.sh

```diff
@@ -123,11 +123,19 @@ callerIdentityJson=$(${AWS_CLI_BIN} sts get-caller-identity)
 ACCOUNT_ID=$(echo "${callerIdentityJson}" | jq -r ".Account")
 MY_IAM_ARN=$(echo "${callerIdentityJson}" | jq -r ".Arn")
 
-# Check whether the AWS CLI v1.19.28/v2.1.30 or later exists
+# Check task existence
 describedTaskJson=$(${AWS_CLI_BIN} ecs describe-tasks \
   --cluster "${CLUSTER_NAME}" \
   --tasks "${TASK_ID}" \
   --output json)
+existTask=$(echo "${describedTaskJson}" | jq -r ".tasks[0].taskDefinitionArn")
+if [[ "x${existTask}" = "xnull" ]]; then
+  printf "${COLOR_RED}Pre-flight check failed: The specified ECS task does not exist.\n\
+Make sure the parameters you have specified for cluster "${CLUSTER_NAME}" and task "${TASK_ID}" are both valid.\n"
```

Comment:
Could you escape the double quotes in this line like `...\"${CLUSTER_NAME}\" and task \"${TASK_ID}\"...`? I think it would be much easier to read for users 😃

Reply:
Just as you said!
I wasn't thinking about end-user readability.
Thank you for your comment.